src/views/EditProfile.js

```diff
@@ -0,0 +1,240 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable jsx-a11y/label-has-for */
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { fieldRemover } from "../helpers/helpers";
+import Input from "../components/common/Inputs/input";
+import BasicButton from "../components/common/Buttons/BasicButton";
+import Message from "../components/common/Message/message";
+import forbidden from "../assets/img/403.png";
+import mainLogo from "../assets/img/main-logo.svg";
+import defaultImage from "../assets/img/defaultImage.png";
+import {
+  fetchCurrentUser,
+  handleFormInput,
+  saveUpdatedUser,
+  uploadImage
+} from "../redux/actions/profileActions";
+
+export class EditProfile extends Component {
+  componentDidMount() {
+    const {
+      match: {
+        params: { username }
+      }
+    } = this.props;
+    const { onFetchProfile } = this.props;
+    onFetchProfile(username);
+  }
+
+  handleChange = ({ target: { name, value } }) => {
+    const { onInputChange } = this.props;
+    onInputChange({ field: name, value });
+  };
+
+  handleImageChange = ({ target }) => {
+    const { profile, onUploadImage, onSaveUpdatedUser } = this.props;
+    onUploadImage(target.files[0]).then(response => {
+      onSaveUpdatedUser(
+        fieldRemover({ ...profile, image: response.secure_url })
+      );
+    });
+  };
+
+  handleSubmit = () => {
+    const { profile, onSaveUpdatedUser } = this.props;
+    onSaveUpdatedUser(fieldRemover(profile));
+  };
+
+  render() {
+    const {
+      profile: { email, firstName, lastName, bio, phone, address, image },
+      error,
+      message,
+      loading,
+      loggedInUser
+    } = this.props;
+    const {
+      match: {
+        params: { username }
+      }
+    } = this.props;
+
+    if (!loggedInUser) {
+      return <img className="loading" src={forbidden} alt="Forbidden" />;
+    }
+    if (loggedInUser) {
```

Comment:
can you combine these two blocks of if statement into one block?  ( just to make it dry)